src/components/ReactComponent/error/404-Compoent.tsx

```
@@ -1,130 +1,192 @@
-                duration: 1.5,
-                repeat: Infinity,
-                ease: "easeInOut",
-              }}
-              className="text-xl"
-            >
-              ←
-            </motion.span>
-          </motion.a>
-        </motion.div>
+            <div>// TODO: Fix broken links</div>
+            <div>// DEBUG: Route not found</div>
```

Biome (1.9.4)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)